src/index.js

```diff
@@ -2,15 +2,15 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import { Provider } from 'react-redux';
 import store from './redux/configureStore';
-import './index.css';
+import './Index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <Provider store={store}>
-      <app />
+      <App />
     </Provider>
```

Comment:
- [ ] Dear @mohamedgamaleldin999999, I noticed you have made some changes to your project. Yet, The tests are still falling. This has to do with some packages/modules that are not available inside the test environments. As stated in the test results, you are missing `module 'enzyme-adapter-react-16' from 'src/__tests__/City.test.js'`, `module 'enzyme-adapter-react-16' from 'src/__tests__/CityList.test.js'`, `module 'react-test-renderer' from 'src/__tests__/Nav.test.js'` and `module 'enzyme-adapter-react-16' from 'src/__tests__/Cities.test.js'`, Please work on this tests and resubmit the project for another review. 

**In the case the test is running well on your computer, Please use a separate machine, clone the project, and try it there.**

![Screen Shot 2023-09-09 at 9 56 21 PM](https://github.com/mohamedgamaleldin999999/metrics-app/assets/69077061/85a26a9e-c6ae-4af3-b18b-709d07b32595)